AppReceiptValidator/AppReceiptValidator/Receipt.swift

```diff
@@ -241,16 +288,40 @@ private struct StringFormatter {
     func format(_ data: Data?) -> String {
         guard let data = data else { return fallback }
 
-        return data.base64EncodedString()
+        return quoted(data.base64EncodedString())
     }
 
     func format(_ date: Date?) -> String {
         guard let date = date else { return fallback }
 
-        return AppReceiptValidator.asn1DateFormatter.string(from: date)
+        return quoted(AppReceiptValidator.asn1DateFormatter.string(from: date))
     }
 
     func format(_ string: String?) -> String {
-        return string ?? fallback
+        return string.map(quoted) ?? fallback
```

Comment:
nit: not sure if the explicit `self` convention is used in this project - if yes, it should be changed to `string.map(self.quoted)` as it not a free function

Reply:
In this project not pervasively. I think we should update throughout the project at one point instead.